constants.js

```diff
@@ -121,8 +121,8 @@ const constants = {
     // for external backends, don't call unless at least 1 minute
     // (60,000 milliseconds) since last call
     externalBackendHealthCheckInterval: 60000,
-    versioningNotImplBackends: { azure: true },
-    mpuMDStoredExternallyBackend: { aws_s3: true },
+    versioningNotImplBackends: { azure: true, gcp: true },
+    mpuMDStoredExternallyBackend: { aws_s3: true, gcp: true },
```

Comment:
Are we actually storing MD on GCP like we do for AWS S3? Because for AWS we do initiate mpu but for GCP we emulate MPU.

Reply:
Yes. On initiate MPU, a 0-byte object will be created that will hold the metadata for the final object.